### DISCUSSION TITLE
episode/04

### DISCUSSION THREAD

**Discussion by giscus** [opensource-f2f/website#63] [General]
# episode/04

技术播客节特别节目：请不要忽视开源中的 Burn out 问题

https://osf2f.net/episode/04

**Comment by xiaowangxiaowang256256** [comment]
我在工作中使用过DedeCMS织梦系统，来进行网站文章的发布与管理。它帮助过我，我很感谢它。 

《关于开源中的 Burn out （精力耗竭）问题，关于可持续性的问题的，一些思考》

1、从开源社区参与者的加入到离开，这是一个完整的生命周期。Burnout是这个生命周期的休止符，可能重启，也可能一直停止。Burnout主要分为两种情况：不幸导致的被动离开，精神层面上的主动离开。被动离开或主动离开，并**不会消耗掉参与者之前的贡献**，仅代表着参与者的贡献暂停了。我们关注的重点，应该是如何促进**开源项目的正向发展**。

2、开源项目的正向发展，需要依赖其**理念**、**人力**、**金钱**。开源项目的理念传播、社区组织、代码贡献都需要人力参与进来。在传统商业项目中，主要是由金钱来驱动，来雇佣人力进行劳动，从而促成人力参与到项目建设中来。而在开源项目中，则依赖更纯粹的**理念驱动**。而金钱则作为补充性的条件。参与者更注重的是，个人精神上的激励，个人能力上的提升，次之是物质上的回报。开源项目的理念越强大，则这份驱动力也会越强大，从而获得更多人力来**驱动项目建设**。

3、总结：Burn out仅代表着一名特定社区成员的离开，若要保持可持续性发展不能过于依赖一个具体的人，而需要将关注点放到**开源项目理念传播与建设**上来。

**Reply by LinuxSuRen** [reply]
从参与者的角度来看，合理地利用自己的精力也是非常值得关注，不管是参与开源还是参与其他事情。与此同时，尽量以高质量的态度与“社区”的人交流，尽量能带给其他人帮助或正能量，避免过度消耗其他人的精力。这算是我对开源 Burn out 的理解。
